### PR TITLE
Import third_party/ouroboros/operator

### PR DESCRIPTION
**Pull request by corona10**
-  Import third_party/ouroboros/operator

### CODE REVIEW

File: compiler/stmt_test.py [hunk 1]
```diff
@@ -104,10 +104,16 @@ def testAugAssignBitAnd(self):
...
-    expected = 'augmented assignment op not implemented'
-    self.assertRaisesRegexp(util.ParseError, expected,
-                            _ParseAndVisit, 'foo **= bar')
+  def testAugAssignPow(self):
+    self.assertEqual((0, '64\n'), _GrumpRun(textwrap.dedent("""\
+        foo = 8
+        foo **= 2
+        print foo""")))
+
+#  def testAugAssignUnsupportedOp(self):
```
Comment (corona10) [inline compiler/stmt_test.py]: @trotterdylan 
Do you have any idea to replace this test?

File: third_party/ouroboros/test/test_operator.py [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+        self.assertEqual(operator.iand     (c, 5), "iand")
+        self.assertEqual(operator.ifloordiv(c, 5), "ifloordiv")
+        self.assertEqual(operator.ilshift  (c, 5), "ilshift")
+        self.assertEqual(operator.imod     (c, 5), "imod")
+        self.assertEqual(operator.imul     (c, 5), "imul")
+        self.assertEqual(operator.ior      (c, 5), "ior")
+        self.assertEqual(operator.ipow     (c, 5), "ipow")
+        self.assertEqual(operator.irshift  (c, 5), "irshift")
+        self.assertEqual(operator.isub     (c, 5), "isub")
+        #self.assertEqual(operator.itruediv (c, 5), "itruediv")
```
Comment (corona10) [inline third_party/ouroboros/test/test_operator.py]: Not yet implemented in grumpy.

**Comment by corona10** [comment]
@trotterdylan 
With the addition of this unit test, we will be able to figure out what else to implement in grumpy.
PTAL

**Comment by corona10** [comment]
@trotterdylan 
The failure of this CI is not related to this PR.

**Comment by corona10** [comment]
@trotterdylan Can I get review for this PR?

File: compiler/stmt_test.py (hunk 1 above)
Comment (trotterdylan) [inline compiler/stmt_test.py]: I'm not sure there are any more unsupported AugAssign operators, so you can just remove it.

**Comment by corona10** [comment]
@trotterdylan Done! PTAL

**Pull request merged**